src/views/CountryThreeWProjects/index.tsx

```diff
@@ -0,0 +1,515 @@
+                        />
+                    </div>
+                </div>
+            )}
+            <Container
+                heading={strings.threeWOngoingProjectsTitle}
+                withHeaderBorder
+                childrenContainerClassName={styles.content}
+                filters={(
+                    <Filter
+                        value={filters}
+                        onChange={setFilters}
+                        districtList={districtList}
+                    />
+                )}
+                actions={(
+                    <>
+                        <Button
+                            variant="primary"
+                            name={undefined}
+                            onClick={handleExportClick}
+                            icons={(<DownloadFillIcon />)}
+                        >
+                            {strings.exportProjects}
+                        </Button>
+                        <Link
+                            to={`/three-w/all/?country=${countryResponse?.iso}`}
+                            withForwardIcon
+                        >
+                            {strings.viewAllProjects}
+                        </Link>
+                    </>
+                )}
+            >
+                <div className={styles.topSection}>
+                    <div className={styles.mapContainer}>
+                        <Map
+                            className={styles.mapContainer}
+                            projectList={ongoingProjects}
+                            districtList={districtList}
+                        />
+                    </div>
+                    <Container
+                        className={styles.sidebar}
+                        heading={strings.threeWInCountryMapSidebarTitle}
+                    >
+                        {Object.entries(districtGroupedProject).map(([districtId, projects]) => {
+                            if (isNotDefined(projects) || projects.length === 0) {
+                                return null;
+                            }
+
+                            const district = districtList.find((d) => d.id === Number(districtId));
+
+                            if (isNotDefined(district?.id)) {
+                                return (
+                                    <ExpandableContainer
+                                        key="others"
+                                        heading={resolveToString(
+                                            strings.otherProjects,
+                                            {
+                                                numProjects: projects.length,
+                                            },
+                                        )}
+                                        headingLevel={4}
+                                        initiallyExpanded
+                                    >
+                                        {projects.map((project) => (
```

Comment:
Let's add a NOTE that projects will always have at least one element because we are using listToGroupList to get this value